numpy/ma/core.pyi

```diff
@@ -371,7 +374,13 @@ class _extrema_operation(_MaskedUFunc):
     def reduce(self, target, axis=...): ...
     def outer(self, a, b): ...
 
-def min(obj, axis=..., out=..., fill_value=..., keepdims=...): ...
+def min(
+    obj: MaskedArray[_ShapeLike, _SCT] | _ArrayLike[_SCT],
```

Comment:
Do you need `_ArrayLike[_SCT]`?  Isn't this file specifically `MaskedArray` (asking b/c I'm not sure).